app/controllers/application_controller.rb

```diff
@@ -9,5 +9,11 @@ def set_locale
     I18n.locale = ::Configuration.locale
   rescue I18n::InvalidLocale => e
     logger.warn "I18n::InvalidLocale #{::Configuration.locale}: #{e.message}"
+  rescue NoMethodError => e
+    if ! ::Configuration.locales_root.exist?
+      logger.warn "I18n failure: configured locale root #{::Configuration.locales_root} does not exist"
+    else
+      raise e
+    end
```

Comment:
I guess this should be added to all of the Rails apps that have `set_locale`?

Also, is this implementation preferable to:

```ruby
def set_locale
  if ::Configuration.locales_root.exist?
    I18n.locale = ::Configuration.locale
  else
    logger.warn "I18n failure: configured locale root #{::Configuration.locales_root} does not exist"
  end
rescue I18n::InvalidLocale => e
  logger.warn "I18n::InvalidLocale #{::Configuration.locale}: #{e.message}"
end
```

Reply:
I need to update this to be a work in progress. At the end of yesterday I was no longer able to duplicate the problem. I want to be sure that this is an actual defect, and not me doing something odd.